Kickstarter-iOS/Views/Cells/BackingCell.swift

```diff
@@ -4,13 +4,29 @@ import Prelude
 import ReactiveExtensions
 import UIKit
 
+internal protocol BackingCellDelegate: class {
+  /// Call when should navigate to Backing Info.
+  func backingCellGoToBackingInfo()
+}
+
 internal final class BackingCell: UITableViewCell, ValueCell {
   fileprivate let viewModel: BackingCellViewModelType = BackingCellViewModel()
 
-  @IBOutlet fileprivate weak var deliveryLabel: UILabel!
-  @IBOutlet fileprivate weak var pledgedLabel: UILabel!
-  @IBOutlet fileprivate weak var rewardLabel: UILabel!
-  @IBOutlet fileprivate weak var rootStackView: UIStackView!
+  @IBOutlet private weak var backingInfoButton: UIButton!
```

Comment:
thanks for switching these back to `private` :/ the migrator was waaaaay too aggressive on making everything `fileprivate`. hopefully they just get rid of `fileprivate` entirely in the next version of swift